src/Components/v2/Carousel.tsx

```diff
@@ -143,7 +144,6 @@ export const SmallCarousel: React.FC<CarouselProps> = props => {
         wrapAround: false,
         ...props.options,
       }}
-      {...props}
```

Comment:
Here we were overriding the option property. By fixing this, we able to use the Carousel component as is (which is responsive) and make the logic in the collection hub rails much simpler.